app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+
+  has_many :likes, dependent: :destroy
+
+  def update_posts_counter
+    author.postsCounter += 1
+    author.save
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the `update_posts_counter` :clap: However it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` or `after_create_commit` to call the `#update_posts_counter` method every time a new post has been saved to database, then make the `update_posts_counter` method private.
  As suggested here :point_down: 
```suggestion
  after_save :update_posts_counter
  
  def recent_comments
    comments.order(created_at: :desc).limit(5)
  end
  
  private
  
  def update_posts_counter
    author.postsCounter += 1
    author.save
  end

```